### PR TITLE
Tiffany/group route

### PR DESCRIPTION
**Pull request by tifli**
## Describe your changes:
Added routes for create and get groups.

## Proof that your changes work (include screenshots if frontend):
Screenshot 2023-04-27 at 9 51 24 PM

## Trello Issue ticker link:
backend

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have DM'd the group chat that I have made a PR and need reviewers.
- [x] I have moved the Trello issue to "Under Review"

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -32,8 +33,19 @@ def create_user():
 
 @app.route('/get-users', methods = ['GET'])
 def get_users():
-    users = User.objects
+    users = User.objects()
```
Comment (TylerAutonSmith) [inline backend/app.py:36]: smart

File: backend/app.py [hunk 2]
```diff
@@ -32,8 +33,19 @@ def create_user():
...
     return jsonify(users)
 
+@app.route('/create-group', methods = ['POST'])
+def create_group():
+    body = request.get_json()
+    new_group = Group(**body).save()
+    return jsonify(new_group)
+
+@app.route('/get-groups', methods = ['GET'])
+def get_group():
```
Comment (TylerAutonSmith) [inline backend/app.py:46]: wow!

**Review by TylerAutonSmith — APPROVED** [review]
Overall hella solid

**Review by joonyoo181 — APPROVED** [review]
LGTM!

**Pull request merged**